bosh-cli.rb

```diff
@@ -0,0 +1,18 @@
+class BoshCli < Formula
+  desc "The new golang bosh cli (alpha)"
+  version "0.0.67"
+  url "https://s3.amazonaws.com/bosh-cli-artifacts/bosh-cli-#{version}-darwin-amd64"
+  sha256 "85d36305c3d9529a8e4756bdf28dfd3c089c787b331ad8afdb673752107e2f8f"
+
+  depends_on :arch => :x86_64
+
+  conflicts_with "pivotal/tap/bosh-cli", :because => "the Pivotal tap ships an older bosh cli version and is outdated"
```

Comment:
@voelzmo,
Yeah it was always installed via rubygems. I will remove this. 
I do however agree with @JesseTAlford, this is not yet the officially supported bosh cli, so we should call that out more obviously and change the binary name so it doesn't conflict with the current cli. Having two things called bosh on the path could lead to strange behavior. I think we should call the formula bosh-cli-alpha, and rename the binary to gosh or gbosh.

@chipchilders apologies, I didn't realize my orgs were set to private, that's very strange as I remember deliberately changing that when I switched to working on an OSS team. Fixed.

Thanks,
Alvaro